modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc

```diff
@@ -175,6 +175,10 @@ if __name__ == '__main__':
 <2> Authenticates with the cluster by using values that you specify when creating the pipeline run
 // Commenting out second part of callout 2 until RHOAIENG-880 is fixed
 //; you can omit this section if the Ray cluster is configured to use the same namespace as the data science project
+[NOTE]
+----
+If your cluster uses self-signed certificates, include `ca-cert-path=__<path>__` in the `TokenAuthentication` parameter list, where `__<path>__` is the path to the cluster-wide Certificate Authority (CA) bundle that contains the self-signed certificates.
```

Comment:
When I used this format for the italics in the release notes, it did not work for me in staging, but it does work elsewhere so maybe it was just the release notes formatting (within the --).

Reply:
I think it should work, I'll fix in a follow-up PR if it doesn't :)